run/events-pubsub/README.md

```diff
@@ -0,0 +1,72 @@
+# Events for Cloud Run – Pub/Sub tutorial
+
+This sample shows how to create a service that processes GCS events
```

Comment:
Add periods.

Reply:
and maybe Pub/Sub events instead of GCS events?